src/sentry/integrations/jira_server/urls.py

```diff
@@ -1,12 +1,13 @@
 from django.urls import re_path
 
 from .search import JiraServerSearchEndpoint
-from .webhooks import JiraIssueUpdatedWebhook
+from .webhooks import JiraServerIssueUpdatedWebhook
 
+# If updating/adding URLs here, make sure to update the JiraServerRequestParser as well
 urlpatterns = [
     re_path(
         r"^issue-updated/(?P<token>[^\/]+)/$",
-        JiraIssueUpdatedWebhook.as_view(),
+        JiraServerIssueUpdatedWebhook.as_view(),
         name="sentry-extensions-jiraserver-issue-updated",
     ),
```

Comment:
I'd like an in-code mechanism to link these two areas of the app: the parsers, and the url patterns, but I haven't put anything together yet. Ideas are welcome, so that future integration engineers don't break hybrid cloud

Reply:
This is a great idea, I'll put this in a ticket to do across all the parsers together